index.html

```diff
@@ -13,53 +13,41 @@
 
 </head>
 <body>
-    <img src="header-background-mobile.png" alt="">
+    <section class="all content">
+        <img src="header-background-mobile.png" alt="">
     
-    <div class="content">
-        <ul>
-            <li class="batery"><img src="images/Capacity.png" alt=""></li>
-            <li class="signal"><img src="images/Cellular Connection.jpg" alt=""></li>
-            <li class="menu"><img src="images/menu.jpg" alt=""></li>
-            <li class="hour"><img src="images/↳ Time.jpg" alt=""></li>
-            <li class="wifi"><img src="images/Wifi.jpg" alt=""></li>
-            <li class="menuu"><img src="images/Union.jpg" alt=""></li>
-        </ul>
-
-    </div>
-        
-    <header class="logo">My Logo
-    </header>
-
-    <br>
-    <br>
-    <div class="contenedor">
+        <nav class="content">
+            <ul>
+                <li class="menuu"><img src="images/Union.jpg" alt=""></li>
+            </ul>
+            <a class="logo" href="index.html">My Logo</a>
+        </nav>   
```

Comment:
- [ ] As suggested previously, kindly wrap logo in the header tags and header tag should also have nav tag which will have hamburger inside it.
Example:
```html
<header>
<nav class="content">
            <ul>
                <li class="menuu"><img src="images/Union.jpg" alt=""></li>
            </ul>
        </nav>  
<a class="logo" href="index.html">My Logo</a> 
</header>
```